### PR TITLE
Update demo for libnetwork ux changes

### PR DESCRIPTION
**Pull request by robbrockbank**
Docs / demo updates from the ux changes for libnetwork.  Untested.

### CODE REVIEW

File: docs/getting-started/default-networking/EnvironmentSetup.md [hunk 1]
```diff
@@ -94,9 +94,9 @@ Check that the hosts have IP addresses assigned, and that your hosts can ping
 one another.
 
 You should also verify each host can access etcd.  The following will return 
-an error if etcd is not available.
+the current etcd version if etcd is not available.
```
Comment (tomdee) [inline docs/getting-started/default-networking/EnvironmentSetup.md]: eeeeeerrrrrr

**Comment by tomdee** [comment]
As just discussed, take a look at the current WIP docker networking docs and link to them as appropriate.

File: docs/getting-started/libnetwork/Demonstration.md [hunk 2]
```diff
@@ -34,40 +36,50 @@ You should see output like this on each node
...
- all in the same network so should be able to ping each other.  B and D are in 
- their own networks so shouldn't be able to ping anyone else.
+all in the same network so should be able to ping each other.  B and D are in 
+their own networks so shouldn't be able to ping anyone else.
     
 On calico-01 check that A can ping C and E.
 
-    docker exec workload-A ping -c 4 srvC
-    docker exec workload-A ping -c 4 srvE
+    docker exec workload-A ping -c 4 workload-C
```
Comment (tomdee) [inline docs/getting-started/libnetwork/Demonstration.md]: I don't think hostname lookups will work for containers in different networks. Maybe you can fully qulify them (workload-d.net3) Look in /etc/hosts

File: docs/getting-started/libnetwork/cloud-config/user-data-first [hunk 3]
```diff
@@ -107,7 +83,7 @@ write_files:
     # binary we have stored that is statically linked and usable by CoreOS.
```
Comment (tomdee) [inline docs/getting-started/libnetwork/cloud-config/user-data-first]: These lines are now meaningless...

File: docs/getting-started/libnetwork/vagrant-ubuntu/Vagrantfile [hunk 4]
```diff
@@ -48,22 +48,21 @@ Vagrant.configure(2) do |config|
       # binary we have stored that is stable and usable by Ubuntu.
```
Comment (tomdee) [inline docs/getting-started/libnetwork/vagrant-ubuntu/Vagrantfile]: This stuff is bit wrong now

File: docs/getting-started/libnetwork/vagrant-ubuntu/Vagrantfile [hunk 5]
```diff
@@ -48,22 +48,21 @@ Vagrant.configure(2) do |config|
...
+            args: "--net=host",
+            cmd: "--advertise-client-urls http://#{primary_ip}:4001 "\
+                 "--listen-client-urls http://0.0.0.0:4001 "
+        end
       end
 
-      # Set Docker to use consul for multihost.
-      host.vm.provision :shell, inline: %Q|sudo sh -c 'echo "DOCKER_OPTS=\"--kv-store=consul:#{primary_ip}:8500\"" > /etc/default/docker'|
+      # Set Docker to use etcd for multihost.
+      host.vm.provision :shell, inline: %Q|sudo sh -c 'echo "DOCKER_OPTS=\"--cluster-store=etcd:#{primary_ip}:4001\"" > /etc/default/docker'|
```
Comment (tomdee) [inline docs/getting-started/libnetwork/vagrant-ubuntu/Vagrantfile]: You have the format of cluster-store wrong. You need some slashes. Check where I've set it in the makefile

File: docs/getting-started/libnetwork/vagrant-ubuntu/Vagrantfile [hunk 6]
```diff
@@ -48,22 +48,21 @@ Vagrant.configure(2) do |config|
...
       if i == 1
-        # Download consul and start.
-        host.vm.provision :shell, inline: <<-SHELL
-          sudo apt-get install -y unzip
-          curl -L --silent https://dl.bintray.com/mitchellh/consul/0.5.2_linux_amd64.zip -o consul.zip
-          unzip consul.zip
-          chmod +x consul
-          nohup ./consul agent -server -bootstrap-expect 1 -data-dir /tmp/consul -client #{primary_ip} > consul.log &
-        SHELL
+        host.vm.provision :docker do |d|
```
Comment (tomdee) [inline docs/getting-started/libnetwork/vagrant-ubuntu/Vagrantfile]: Hmmmm, I'm not sure that we can use a docker version of etcd if Docker is meant to be using it as a cluster store...

File: docs/getting-started/libnetwork/cloud-config/user-data-others [hunk 7]
```diff
@@ -33,7 +33,8 @@ coreos:
       [Service]
       RemainAfterExit=yes
       Type=oneshot
-      ExecStart=/home/core/get_experimental_docker.sh
+      ExecStart=/home/core/get_latest_docker.sh
```
Comment (tomdee) [inline docs/getting-started/libnetwork/cloud-config/user-data-others]: Where is this file?

File: docs/getting-started/libnetwork/vagrant-ubuntu/Vagrantfile (hunk 6 above)
Comment (robbrockbank) [inline docs/getting-started/libnetwork/vagrant-ubuntu/Vagrantfile]: Yup I suddenly realised that last night as soon as I started testing :-/  oops

File: docs/getting-started/libnetwork/cloud-config/user-data-others (hunk 7 above)
Comment (robbrockbank) [inline docs/getting-started/libnetwork/cloud-config/user-data-others]: yup not required.  was a remnant of an initial change

File: docs/getting-started/libnetwork/vagrant-ubuntu/Vagrantfile (hunk 4 above)
Comment (robbrockbank) [inline docs/getting-started/libnetwork/vagrant-ubuntu/Vagrantfile]: quite right you are.

File: docs/getting-started/libnetwork/vagrant-ubuntu/Vagrantfile (hunk 5 above)
Comment (robbrockbank) [inline docs/getting-started/libnetwork/vagrant-ubuntu/Vagrantfile]: cool - thx

**Comment by tomdee** [comment]
@robbrockbank you gonna close this?

**Comment by robbrockbank** [comment]
Obsolete.  Now included in release branch

**Pull request closed without merging**